### ISSUE TITLE
Is --antialias is actually working ?

### ISSUE THREAD

**Issue by kimstik** [gerbv/gerbv#222]
I see that GIU allows you to specify the render quality in a range {fast, xor, normal, high}, which works quite well.
How can I specify this correctly from the command line?

**Comment by rampageservices** [comment]
Yes, `--antialias` / `-a` works for CLI export. It switches the render type from `CAIRO_NORMAL` to `CAIRO_HIGH_QUALITY`, which enables smoothest-edge rendering in the exported image (PNG, PDF, SVG, PS).

The GUI exposes 4 render modes (GDK fast, GDK XOR, Cairo normal, Cairo high quality), but the CLI only offers the binary toggle because the two GDK modes are for interactive display — they aren't applicable to file export, which always uses Cairo.

Usage:
```
gerbv -a --export=png -o output.png input.gbr
```

Without `-a`, export uses Cairo normal. With `-a`, it uses Cairo high quality.

**Comment by kimstik** [comment]
Tested on Windows with the binaries below --antialias has no effect on the GUI render mode. The window opens in Fast regardless of the flag.

gerbv.exe --units=mm --antialias

Render mode stays on Fast in the View menu. Changing it manually in the GUI works fine, so the modes themselves are functional -the flag just doesn't apply on startup.

Reproduced with:

https://github.com/gerbv/gerbv/releases/download/v2.11.0/gerbv_v2.11.0_.Windows.amd64.zip
https://gerbv.github.io/ci/gerbv_2025-06-11_180c67_(Windows%20amd64).zip

Haven't tested whether the export-only path (-x png -a) is also affected.
